### DISCUSSION TITLE
Updating without creating new empty volume

### DISCUSSION THREAD

**Discussion by riccardospeggiorin-centropaghe** [containrrr/watchtower#813] [Q&A] [answered]
Hi, I am using watchtower to update unify controller image on a QNAP NAS.
The process is working flawlessy but I see that when it restarts the new container with new downloaded container, it creates new empty volume.
I have to manually copy the contents from the old one to the new for not losing data.
It's possible to forse it to use the old volumes?
Thanks

**Comment by piksel** [comment, marked as answer]
How are you starting your container?

I assume you are using an image that has a VOLUME in it's Dockerfile, but you are not explicitly mounting said volume when running?
If so, you can create a docker volume and specify it using `-v VOLUME_NAME:MOUNT_POINT` when running the container.

**Reply by riccardospeggiorin-centropaghe** [reply]
I cannot quite answer to this question, cause I have done it using the QNAP gui. I see this on the gui
![image](https://user-images.githubusercontent.com/66776629/107350049-3b40b800-6ac9-11eb-9f79-e0f500f98a69.png)


# docker volume ls
DRIVER              VOLUME NAME
local               2bb21f8147f6cb97978078a42e0d31b063c5cde17066c30f34a89369df928dc0
local               4479687c0ab2f469c25ace45ff93015bde3a1880a2efbfcab9c911efec9d0226

**Reply by piksel** [reply]
I think that means what I presumed was happening. Watchtower itself cannot do much about this unfortunately, but I have no experience in setting up docker on a QNAP device.
